packages/core/src/common/utils.ts

```diff
@@ -24,6 +24,23 @@ export function isFunction(value: any): value is Function {
     return typeof value === "function";
 }
 
+/**
+ * Returns true if `node` is null/undefined, false, empty string, or an array
+ * composed of those. If `node` is an array, only one level of the array is
+ * checked, for performance reasons.
+ */
+export function isEmptyReactNode(node?: React.ReactNode, skipArray = false): boolean {
```

Comment:
`isEmptyReactNode`/`isReactNodeEmpty`? Would normally lean toward the second. The former implies to me that the function is also going to verify that the parameter _is_ a `ReactNode` in the first place.